runironic-inspector.sh

```diff
@@ -5,6 +5,12 @@ PROVISIONING_INTERFACE=${PROVISIONING_INTERFACE:-"provisioning"}
 CONFIG=/etc/ironic-inspector/inspector.conf
 PROVISIONING_IP=$(ip -4 address show dev "$PROVISIONING_INTERFACE" | grep -oP '(?<=inet\s)\d+(\.\d+){3}' | head -n 1)
 
+until [ ! -z "${IRONIC_IP}" ]; do
```

Comment:
This should be PROVISIONING_IP to match the variable name above

Reply:
thanks